tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+    if (!response.ok) {
+        throw new Error("Failed");
+    }
+    const data = await response.text();
+    return data;
+}
+"""
+
+
+def run_tldr(args: list[str], cwd: str = None) -> dict:
+    """Run tldr command and return parsed JSON output."""
+    cmd = ["tldr"] + args + ["--lang", "typescript"]
```

Ruff (0.14.11)

Consider `["tldr", *args, "--lang", "typescript"]` instead of concatenation

Replace with `["tldr", *args, "--lang", "typescript"]`

(RUF005)